crates/sui-json/src/tests.rs

```diff
@@ -460,9 +460,15 @@ fn test_basic_args_linter_top_level() {
     .map(|q| SuiJsonValue::new(q.clone()).unwrap())
     .collect();
 
-    let json_args =
-        resolve_move_function_args(example_package, module.clone(), function.clone(), &[], args)
-            .unwrap();
+    let json_args = resolve_move_function_args(
+        example_package,
+        module.clone(),
+        function.clone(),
+        &[],
+        args,
+        false,
```

Comment:
nit: can you annotate the boolean with the parameter name? (Also a couple more cases below).

```suggestion
        /* allow_arbitrary_functional_call */ false,
```